src/controllers/tripsController.js

```diff
@@ -131,6 +125,21 @@ class tripsController {
       errorResponse(res, 500, error.message);
     }
   }
+
+  static async getTrips(req, res) {
+    const { userID } = req.user;
+    try {
+      const trips = await travelRequests.findAll({
+        where: { userID },
+        attributes: {
+          exclude: ['id', 'userID', 'passportNumber', 'gender', 'createdAt', 'updatedAt']
+        },
+      });
+      return successResponse(res, 200, req.i18n.__('retrievedSuccessfully'), trips);
+    } catch (error) {
+      return errorResponse(res, error.status || 500, error.message);
```

Comment:
```suggestion
      return errorResponse(res, error.status || 500, setLanguage(req.user.preferedLang).__('internalServerError'));
```
Would you please consider returning a specific message to a user other than server default errors?
The locale for ```internalServerError``` key has been already set.

Reply:
That sounds better